authlib/auth.py

```diff
@@ -14,10 +15,15 @@
 ENC_NONCE = osenv.get('ENC_NONCE')
 
 STORAGE = osenv.get('STORAGE', 'SQLITE')
+COOKIE_NAME = osenv.get('COOKIE_NAME')
 store = None
 
 # ------------------------------------------------------------------------------
 
+@st.cache(allow_output_mutation=True)
+def get_manager():
```

Comment:
I'd create a `CookieManager` object with CRUD operations. You could define that in this file or in a separate `authlib/common/cookie_manager.py` file, and import it here. (Remember to add the reference in `authlib/__init__.py`.)